.github/workflows/build_prs.yaml

```diff
@@ -19,5 +19,4 @@ jobs:
           cache: 'yarn'
       - run: yarn install --frozen-lockfile --ignore-scripts
       - run: yarn lint
-      - run: yarn build:backend
-      #- run: yarn build:frontend # not needed
+      - run: yarn build
```

Comment:
Not sure if this is intended, but build will also build the frontend, in which case we need to remove --ignore-scripts from yarn install to also install frontend dependencies